taskcluster/ci/browsertime/kind.yml

```diff
@@ -232,3 +232,38 @@ jobs:
         test-name: google-search-restaurants
         treeherder:
             symbol: 'Btime(tp6m-28-c)'
+
+    youtube-playback:
+        test-name: browsertime-youtube-playback
+        treeherder:
+            symbol: 'Btime(ytp)'
+
+    youtube-playback-av1-sfr:
```

Comment:
Remove all of the 2020 variants for now, we'll add those later when they've been verified.

Reply:
2020 variants meaning all except `youtube-playback`?

Reply:
yup that's right